include/quda_milc_interface.h

```diff
@@ -40,6 +40,8 @@ extern "C" {
     int make_resident_solution; /** Make the solution resident and don't copy back */
     int use_resident_solution; /** Use the resident solution */
     QudaInverterType solver_type; /** Type of solver to use */
+    double tadpole; /** Tadpole improvement factor */
```

Comment:
I'd add a comment saying that for HISQ fermions in particular this should be set to 1.0 b/c the tadpole coefficient for the original gauge fields gets baked into the fat/long link construct.